osm2pgsql.lua

```diff
@@ -499,128 +499,116 @@ function osm2pgsql.process_node(object)
         object.tags.wikidata == 'Q994730') then
         output_hstore['disputed_by'] = 'CN;RU;IN;GR'
     end
--- Recast Northern Cyprus as country label and turn off for several POVs including China and Russia
+
+-- Turn off Northern Cyprus label
     if object.tags.place and object.tags.wikidata == 'Q23681' then
-        output_hstore['place'] = 'country'
-        output_hstore['disputed_by'] = 'CN;RU;IN;GR;CY'
+        output_hstore['place'] = nil
     end
--- Turn off Abkhazia label for most countries
+-- Turn off Abkhazia label
     if object.tags.place and object.tags.wikidata == 'Q23334' then
-        output_hstore['place:AR'] = 'region'
```

Comment:
I think you want to make the default place tag `region`, and only have a override for `place:RU` here.